### DISCUSSION TITLE
Keyball46のファームウェアの最新のソースコードはどこを見ればよいですか?

### DISCUSSION THREAD

**Discussion by koron** [Yowkees/keyball#12] [Q&A] [answered]
どちらのソースが最新のファームウェアでしょうか?

1. このレポジトリの qmk_firmware/ ディレクトリにあるやつ
2. [qmk_fimwareのフォークのupdate_keyball46_documentブランチ](https://github.com/Yowkees/qmk_firmware/tree/update_keyball46_document)
3. [qmk_firmwareのフォークのupdate_keymaps_documentブランチ](https://github.com/Yowkees/qmk_firmware/tree/update_keymaps_document)
4. その他(URL等を教えていただければ)

**Comment by Yowkees** [comment, marked as answer]
1番が最新です！

**Comment by koron** [comment]
ありがとうございます。
